bundle/manifests/netobserv-operator.clusterserviceversion.yaml

```diff
@@ -1335,8 +1333,7 @@ spec:
   version: 1.0.5
   webhookdefinitions:
   - admissionReviewVersions:
-    - v1beta1
-    - v1beta2
+    - v1
```

Comment:
why you have this change ?

Reply:
not intentional, it was generated. I opened a bug: https://issues.redhat.com/browse/NETOBSERV-1665.

Reply:
I restarted the bundle check , this time it passed

Also: I've a fix here: https://github.com/netobserv/network-observability-operator/pull/662